src/components/Media.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { apiKey, domainName } from '../utils/apiUrls';
+import { posterUrlBig } from '../utils/apiUrls';
+import MediaPage from './media_page/MediaPage';
+import averageColor from '../utils/averageColor';
+import Loading from './Loading';
+const setStyle = document.documentElement.style;
+
+export default function Movie() {
+    // Requiring information about the media
+    const location = useLocation();
+    const data = location.state;
+    const { mediaType, id} = data;
```

Comment:
Details:
```suggestion
    const { mediaType, id } = data;
```
You could use prettier or ES-Lint to format this on save.